### ISSUE TITLE
Hephaestus (Tinkers' Construct's fabric port) axes don't trigger the chop

### ISSUE THREAD

**Issue by ChromaPIE** [hammertater/treechop#271]
Don't know if it's actually a problem on Heph's side but the axes in the mod break log blocks in the vanilla way.

**Comment by hammertater** [comment]
Last time I looked at it there wasn't much I could do, but I see there's been updates so maybe there's hope. I'll look into it

**Comment by MiguelMoita** [comment]
I'm having the same problem in 1.20.1, was there any thing made to correct it?

**Comment by hammertater** [comment]
Currently, I think the best thing you can do is add their axes/etc. to treechop's chopping tools blacklist. For better compatibility, it would be best if the Hephaestus devs could support treechop (using treechop's API). Doing it on my end would require me to intrusively change how their stuff works, which isn't ideal for many reasons.

**Comment by MiguelMoita** [comment]
![Image](https://github.com/user-attachments/assets/2c239f18-74f5-4ea2-bc65-d0628f371c71)
So I just need to put "hephaestus: itemName" , for every axe/etc?

**Comment by hammertater** [comment]
Yes, but no space. It's possible they use something like "tinkersconstruct" instead of "hephaestus". You can show item IDs in game by pressing F3+H, I think. Let me know if it doesn't work right and I'll look into it
